packages/relay-sdk/src/index.ts

```diff
@@ -0,0 +1,151 @@
+     * and the the payer secret is known.
+     * 
+     * ```typescript
+     * const client = new LitRelayClient('https://habanero.lit.dev', 'my-api-key', 'my-payer-secret');
+     * ```
+     * 
+     * @param baseUrl
+     * @param apiKey
+     * @param payerSecret
+     */
+    private constructor(baseUrl: string, apiKey: string, payerSecret: string) {
+        this.baseUrl = baseUrl;
+        this.apiKey = apiKey;
+
+        if (payerSecret) {
+            this.payerSecret = payerSecret;
+        }
+    }
+
+    /**
+     * Adds a new payee to the payer's delegation list.
+     * 
+     * ```typescript
+     * const client = await LitRelayClient.connect('habanero', 'my-api-key', 'my-payer-secret');
+     * const result = await client.addPayee('payee-wallet-address');
+     * ```
+     * @param payeeAddress
+     * 
+     * @returns Promise<{
+     *  success: false;
+     *  error: string;
+     * } | {
+     *  success: true;
+     *  tokenId: string;
+     * }>
+     */
+    public async addPayee(payeeAddress: string) {
+        if (!this.payerSecret) {
+            throw new Error('Payer secret not set');
+        }
+
+        const res = await fetch(`${this.baseUrl}/add-users`, {
+            method: 'POST',
+            headers: {
+                'Content-Type': 'application/json',
+                'api-key': this.apiKey,
+                'payer-secret-key': this.payerSecret,
+            },
+            body: JSON.stringify([payeeAddress]),
+        });
+
+        if (res.status !== 200) {
+            return {
+                success: false,
+                error: 'Failed to add payee: request failed',
+            };
+        }
+
+        const data = await res.json();
+
+        return {
+            success: true,
+            tokenId: data.tokenId,
+        };
+    }
+
+    /**
+     * Registers a new payer with the Lit Relay server using the provided API key. Returns
+     * a new LitRelayClient instance with the payer secret key.
+     * 
+     * ```typescript
+     * const client = await LitRelayClient.register('habanero', 'my-api-key');
+     * ```
+     * 
+     * @param network 
+     * @param apiKey 
+     * 
+     * @returns LitRelayClient
+     */
+    public static async register(network: SupportedNetworks, apiKey: string) {
+        if (network !== 'habanero' && network !== 'manzano') {
+            throw new Error('Invalid network');
+        }
+
+        const baseUrl = getRelayURLByNetwork(network);
+        const res = await fetch(`${baseUrl}/register-payer`, {
+            method: 'POST',
+            headers: {
+                'Content-Type': 'application/json',
+                'api-key': apiKey,
+            },
+        });
+
+        if (res.status !== 200) {
+            throw new Error('Failed to register payer: request failed');
+        }
+
+        const data = await res.json();
+
+        if (!data.payerSecretKey) {
```

Comment:
If the API is going to return `payerSecretKey`, can we name it the same thing on this object? It'd be nice for the same magic strings to be used in both the client and server contexts for easier discoverability